### PR TITLE
[Teacher][MBL-11706] Post policy settings view, post/hide grades

### PR DESCRIPTION
**Pull request by CalvinKern**

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1005225&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

File: apps/teacher/src/main/res/layout/adapter_post_policy_section.xml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (C) 2019 - present Instructure, Inc.
+  ~
+  ~     Licensed under the Apache License, Version 2.0 (the "License");
```
Comment (JordanMarshall) [inline apps/teacher/src/main/res/layout/adapter_post_policy_section.xml]: GPL for non-test app code

File: apps/teacher/src/main/res/layout/dialog_post_graded_everyone.xml [hunk 2]
```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (C) 2019 - present Instructure, Inc.
+  ~
+  ~     Licensed under the Apache License, Version 2.0 (the "License");
```
Comment (JordanMarshall) [inline apps/teacher/src/main/res/layout/dialog_post_graded_everyone.xml]: GPL

File: apps/teacher/src/main/res/layout/fragment_post_grade.xml [hunk 3]
```diff
@@ -0,0 +1,186 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (C) 2019 - present Instructure, Inc.
+  ~
+  ~     Licensed under the Apache License, Version 2.0 (the "License");
```
Comment (JordanMarshall) [inline apps/teacher/src/main/res/layout/fragment_post_grade.xml]: GPL

**Pull request merged**